docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md

```diff
@@ -484,15 +484,15 @@ settings:
 
 ### Provision notification policies
 
-Create or reset notification policies in your Grafana instance(s).
+Create or reset the notification policy tree in your Grafana instance(s).
```

Comment:
@alexweav do we want to add a sentence or two, that only one policy per org is possible and to use sub-routing?

Reply:
I added one in the root page: https://github.com/grafana/grafana/blob/c678d80a0adac155a84f86cadb26509621c5d06f/docs/sources/alerting/set-up/provision-alerting-resources/_index.md?plain=1#L33

Maybe we add it everywhere?

Reply:
I think I would rather add it here, as you could easily miss it when you are directly going here. I let you decide.